terraform-runner/README.MD

```diff
@@ -33,3 +33,9 @@ as you can see in the following:
 docker pull ghcr.io/meshcloud/unipipe-terraform-runner:latest
 docker run -t -i -e GIT_SSH_KEY="$(<my-ssh-key-file)" --env-file env.list ghcr.io/meshcloud/unipipe-terraform-runner:latest
 ```
+
+# Configuration Reference
```

Comment:
d: Why did you put this to an extra block in the README? We already define other configurations above. I think we should describe it similar to the SSH key, with the difference that it is optional to define it. But all in all it does not seem like a totally different env variable, compared to those we mentioned above. And especially, as it must be a file with line breaks, it makes sense to use the same syntax as for the SSH key, right?